ext/CUDAExt.jl

```diff
@@ -50,7 +50,7 @@ function kern_findmax!(input::CuDeviceMatrix{T}, idst::Integer, idend::Integer)
     return
 end
 
-function maxpool(x::CuArray; siz = 2, stride = 1)
+function maxpool(x::CuArray; siz = 2, stride = 1, kernel = 2)
```

Comment:
Is this needed/intentional?

Reply:
Yeah. Fixed a bug on gpu I introduced when making this ext. `kernel` was undefined.